pkg/cmd/server/start/master_args.go

```diff
@@ -241,16 +241,7 @@ func (args MasterArgs) BuildSerializeableMasterConfig() (*configapi.MasterConfig
 			Latest: args.ImageFormatArgs.ImageTemplate.Latest,
 		},
 
-		// List public registries that we are allowing to import images from by default.
-		// By default all registries have set to be "secure", iow. the port for them is
-		// defaulted to "443".
-		// If the registry you are adding here is insecure, you can add 'Insecure: true' which
-		// in that case it will default to port '80'.
-		// If the registry you are adding use custom port, you have to specify the port as
-		// part of the domain name.
-		ImagePolicyConfig: configapi.ImagePolicyConfig{
-			AllowedRegistriesForImport: configapi.DefaultAllowedRegistriesForImport,
-		},
+		ImagePolicyConfig: configapi.ImagePolicyConfig{},
```

Comment:
is the constant even used anywhere anymore then? can we get rid of it?

Reply:
in the e2e tests

Reply:
and in the integration tests

Reply:
We could call it recommended.